### PR TITLE
add fragileslave.com url scraper

### PR DESCRIPTION
**Pull request by xxlbug**
Adds support for urls from fragileslave.com (title, date, img, tags, details).

Since it's my first scraper, please review a little bit more carefully :-)

### CODE REVIEW

File: scrapers/FragileSlave.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        selector: //div[@class="videocontent"]/p/text()
+      Date:
+        selector: $details[1]/text()
+        postProcess:
+          - replace:
+              - regex: (\s*\|.*)
+                with:
+              - regex: (\d{2})\/(\d{2})\/(\d{4})
+                with: $3-$1-$2
+          - parseDate: 2020-02-01
```
Comment (bnkai) [inline scrapers/FragileSlave.yml:25]: The selector matches the commas as well. Adding a `[1]` at the  end gets us the correct text only
The parseDate should be `          - parseDate: 2006-01-02` 
Tbf even the second regex is not needed
```yaml
      Date:
        selector: $details[1]/text()[1]
        postProcess:
          - replace:
              - regex: (\s*\|.*)
                with:
          - parseDate: 01/02/2006
 ```
is enough

File: scrapers/FragileSlave.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      Image:
+        selector: //div[@class='trailerblock'][1]/script
+        postProcess:
+          - replace:
+              - regex: .*<img.*src0_3x="(.*\.jpg)".*
+                with: $1
```
Comment (bnkai) [inline scrapers/FragileSlave.yml:31]: In some scenes the `script` is not present `https://www.fragileslave.com/updates/trailers/Strapped-down-and-Climaxing.html` so you get no image
The images you get are too big imho, we should get the 1x or 2x by default
The regex needs to be more specific to work for both cases
The below seems to work ok for me
```yaml
      Image:
        # replace src0_1x in the below lines with src0_2x or src0_3x if you need higher res covers
        selector: //div[@class='trailerblock'][1]/script|//img[@class="update_thumb thumbs stdimage"]/@src0_1x
        postProcess:
          - replace:
              - regex: .*<img.*src0_1x="([^"]+\.jpg)".*
                with: $1
```

File: scrapers/FragileSlave.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      Studio:
+        Name: 
+          selector: //head/title/text()
+          postProcess:
+            - replace:
+                - regex: (^[\w]+ [\w]+) - .*
+                  with: $1
```
Comment (bnkai) [inline scrapers/FragileSlave.yml:38]: Is there any other studio? If not a plain
```yaml
      Studio:
        Name:
          fixed: Fragile Slave
```
should be better

File: scrapers/FragileSlave.yml (hunk 1 above)
Comment (xxlbug) [reply scrapers/FragileSlave.yml]: corrected, thank you for the date suggestion

File: scrapers/FragileSlave.yml (hunk 2 above)
Comment (xxlbug) [reply scrapers/FragileSlave.yml]: streamlined it even more and reduced the used image resolution

File: scrapers/FragileSlave.yml (hunk 3 above)
Comment (xxlbug) [reply scrapers/FragileSlave.yml]: not that i know of, so lets go with your solution :-) Thank you for the review!

**Pull request merged**